StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/VerticalSavedPaymentMethodsViewControllerTests.swift

```diff
@@ -94,7 +94,8 @@ class VerticalSavedPaymentMethodsViewControllerTests: XCTestCase {
                                                                        paymentMethods: singlePaymentMethods,
                                                                        paymentMethodRemove: true,
                                                                        isCBCEligible: false)
-        XCTAssertTrue(viewController.canEdit)
```

Comment:
This test method name was wrong, right? 
`testCanEdit_singleRemovableAndEditablePaymentMethod_returnsTrue`
should be
`testCanEdit_singleRemovableAndNotEditablePaymentMethod_returnsTrue` or something.